src/content/pages/build/taco.md

```diff
@@ -0,0 +1,213 @@
+        icon: /images/iot-icon.svg
+      - title: "Activism & Human Rights"
+        description: "TACo protects the most vulnerable in society – those who may suffer  intimidation, censorship and violence as a consequence of their data being leaked."
+        icon: /images/activism-icon.svg
+tacoDiagramsInfo:
+  title: "Integrate a Privacy Layer with Nothing to Hide"
+  description: "TACo isn't just auditable, it's comprehensible. From high-level schematic to client code."
+  image: /images/taco-grid.svg
+  background: /images/glowing-background.png
+tacoDiagrams:
+  - title: "Generating a DKG Cohort"
+    preTitle: "learn"
+    description: "Independent nodes are sampled from the Threshold Network, forming a cohort that collectively generates a DKG public key."
+    image: /images/first-taco-diagram.svg
+    buttons:
+      - label: "nucypher/ferveo"
+        url: https://github.com/nucypher/ferveo
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher-contracts"
+        url: https://github.com/nucypher/nucypher-contracts
+        variant: EXTERNAL_OUTLINE
+  - title: "Encrypting Private Data"
+    preTitle: "learn"
+    description: "Data owners specify decryption conditions, embedded alongside the private data. The payload is encrypted with the DKG public key."
+    image: "/images/second-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+  - title: "Accessing Decrypted Data"
+    preTitle: "learn"
+    description: "Data requesters present proof to the cohort, who verify condition fulfillment before provisioning them with decryption material."
+    image: "/images/third-taco-diagram.svg"
+    buttons:
+      - label: "taco-web"
+        url: https://github.com/nucypher/taco-web
+        variant: EXTERNAL_OUTLINE
+      - label: "nucypher/main"
+        url: https://github.com/nucypher/nucypher/tree/main
+        variant: EXTERNAL_OUTLINE
+tacoExamplesInfo:
+  title: "Access conditions that fit your use case"
+  preTitle: "build"
+  description: "Whatever the business logic for data access, TACo nodes will enforce it. Specify and combine EVM, RPC and time-based conditions. Non-blockchain coming soon."
+  image: "/images/ecosystem-grid.svg"
+  rowReverse: false
+tacoExamples:
+  - title: "NFT-gating"
+    description: "Predicate decryption rights on the holding of special-purpose NFTs. Works for digital media, event streams and beyond."
+    image: "/images/nft-gating.png"
+  - title: "Secret recovery"
+    description: "Share your seed phrase or password  – to beneficiaries or your future self – without risking DOS by an intermediary."
```

Comment:
```suggestion
    description: "Securely share your seed phrase or password, conditionally with beneficiaries or your future self, without risking DOS by an intermediary."
```

Reply:
```suggestion
    description: "Securely share your seed phrase/password with beneficiaries or your future self, without risking DOS by an intermediary."
```
'Conditionally' feels a bit clumsy here, but happy to remove the hyphens